waku/v2/protocol/waku_rln_relay/rln.nim

```diff
@@ -23,19 +23,40 @@ type Buffer* = object
   `ptr`*: ptr uint8
```

Comment:
What happens if you add .push raises: [Defect].} a la https://github.com/status-im/nim-waku/issues/289 here? Curious if you risk getting a bunch of NullPointerExceptions here

Reply:
I added that to the rln module, it compiled just fine. Do you also want me to add it to all the other procs that involve rln lib procedures?

Reply:
oh nice, I think the general idea is to have it in each module/namespace, but this should be fine too